resolver-dns/src/main/java/io/netty/resolver/dns/DnsResolveContext.java

```diff
@@ -262,7 +262,8 @@ private boolean hasNDots() {
         private static final long serialVersionUID = -8573510133644997085L;
 
         SearchDomainUnknownHostException(Throwable cause, String originalHostname) {
-            super("Search domain query failed. Original hostname: '" + originalHostname + "' " + cause.getMessage());
+            super("Failed to resolve '" + originalHostname + "' and search domain query for configured domain" +
+                    " failed as well. " + cause.getMessage());
```

Comment:
Will a list of search domains be useful in this exception message?

Reply:
Good point... PTAL again